docs/contributing.rst

```diff
@@ -32,5 +32,5 @@ Community Guidelines
 This project follows `Google's Open Source Community
 Guidelines <https://opensource.google/conduct/>`__.
 
-.. include:: code-of-conduct
+.. include:: code-of-conduct.rst
```

Comment:
Are we sure that this needs a `.rst` at the end?

Reply:
Already removed entirely from `open-source-pdks` repository. Which it's not in this pull request now.